common/src/Model/BrushNode.h

```diff
@@ -61,6 +61,7 @@ namespace TrenchBroom {
         private:
             mutable std::unique_ptr<Renderer::BrushRendererBrushCache> m_brushRendererBrushCache; // unique_ptr for breaking header dependencies
             Brush m_brush; // must be destroyed before the brush renderer cache
+            size_t m_selectedFaceCount = 0u;
```

Comment:
Hah I had no idea about this, guess it was added in c++11.

Reply:
Yeah. I actually prefer setting it in the constructor for consistency, but this syntax has the advantage that you only need to put the initialization in one place instead of doing it over and over in multiple constructors.